cli/test-utils/src/util.rs

```diff
@@ -689,6 +689,21 @@ pub fn dir_list<P: AsRef<Path>>(dir: P) -> Vec<String> {
         .collect()
 }
 
+/// Creates a cross-platform remapping string for use in tests
+///
+/// NOTE: This probably should be unnecessary, and remappings should probably
+/// be canonicalized.
+pub fn remapping_str(src: &str, dest: &str) -> String {
+    // NOTE(onbjerg): The `trim_end_matches` is because the path itself on Windows is normalized
+    // except for the last character which is still a /...
+    format!(
+        "{}={}/",
+        src,
+        dest.trim_end_matches('/')
+            .replace('/', std::str::from_utf8(&[std::path::MAIN_SEPARATOR as u8]).unwrap())
+    )
```

Comment:
Remappings from `Config` on Windows seem to use Windows paths, *except* if part of the path is from another remapping (see the config tests for examples).

The remappings we use to compare in tests however are not canonicalized. I'm not sure what the best fix is given how flexible remappings are